### PR TITLE
commonItems update, tweaks

### PR DESCRIPTION
**Pull request by IhateTrains**
- commonItems update, purge of unused
- count -> contains
- Simplified getString and getInts usage
- std::find -> std::ranges::find
- Final classes

### CODE REVIEW

**Review by Idhrendur — APPROVED** [review]
Wait for Zemurin's approval, but looks good to me.

**Review by Zemurin — APPROVED** [review]

**Pull request merged**
